CondaPkg.toml

```diff
@@ -1,3 +1,4 @@
 [deps]
+python = "3.9"
```

Comment:
Attempting to install `numpy` on windows with python 3.10 seems to be [running into this](https://stackoverflow.com/questions/58868528/importing-the-numpy-c-extensions-failed), so staying on 3.9 for now